android/app/src/main/java/app/organicmaps/routing/RoutingInfo.java

```diff
@@ -24,6 +24,8 @@ public class RoutingInfo
   public final String currentStreet;
```

Comment:
Does currentStreet contain the road where the car is moving now?

Reply:
Yes